src/api/authorData.js

```diff
@@ -17,13 +37,33 @@ const getSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 });
 
 // FIXME: DELETE AUTHOR
-const deleteSingleAuthor = () => {};
+const deleteSingleAuthor = (firebaseKey, uid) => new Promise((resolve, reject) => {
```

Comment:
```suggestion
const deleteSingleAuthor = (authorId, uid) => new Promise((resolve, reject) => {
```

It's good to keep your parameters as descriptive as possible. This makes for well-documented code!